google/resource_container_cluster_migrate.go

```diff
@@ -21,6 +21,9 @@ func resourceContainerClusterMigrateState(
 	case 0:
 		log.Println("[INFO] Found Container Cluster State v0; migrating to v1")
 		return migrateClusterStateV0toV1(is)
+	case 1:
```

Comment:
So sorry to do this to you- my coworker pointed out to me that we've been unnecessarily adding state migrations when changing from lists to sets, so this can actually be removed.

Reply:
I have an error on migration test if i remove it

Reply:
Sorry, I meant that you don't need the migration after all.